build/.pa11yci.json

```diff
@@ -8,7 +8,7 @@
       "axe"
     ],
     "timeout": 100000,
-    "hideElements": ".bd-search, [id*='tarteaucitron'], #TableOfContents, .text-primary, .navbar-light .navbar-brand, .accordion-button:not(.collapsed), .active, [aria-current], select:disabled, [disabled] label, [disabled] + label, .modal, .bd-example nav, .badge.rounded-pill.bg-info.text-white, .exclude-from-pa11y-analysis",
+    "hideElements": ".bd-search, [id*='tarteaucitron'], #TableOfContents, .text-primary, .navbar-light .navbar-brand, .accordion-button:not(.collapsed), .active, [aria-current], select:disabled, [disabled] label, [disabled] + label, .modal, .bd-example nav, .badge.rounded-pill.bg-info.text-white, .exclude-from-pa11y-analysis, a.disabled",
```

Comment:
Had to add the `a.disabled` rule in order to avoid pa11y to scream. I'm still wondering it is necessary since the disabled link color hasn't changed.

Reply:
yes, because axe ignore color contrast if aria-disabled=true. Following this update there's no more aria-disabled attribute so the contrast is check. This is a good way to ignore those error as these are disabled elements